src/plugins/blackout/blackout.js

```diff
@@ -95,11 +97,16 @@
         }, false );
 
         gc.addEventListener( document, "keyup", function( event ) {
-            if ( event.keyCode === 66 ) {
+            // b or . -> . is sent by presentation remote controllers
+            if ( event.keyCode === 66 || event.keyCode === 190) {
                 event.preventDefault();
             }
         }, false );
 
+        util.triggerEvent( document, "impress:help:add",
+            { command: "b or .", text: "Blackout", row: 100 } 
+        );
+
```

Comment:
I think this option is useful to show on help. It's easy to forget about that. I think should have an help option that shows all help. It would be easy to track the features and know what its available during the presentation.

Reply:
A full help is useful when publishing the presentation, as the visitors can see the options available even when they don't know impress.js

Reply:
A full help is useful when publishing the presentation, as the visitors can see the options available even when they don't know impress.js

Reply:
Yes, but a long help is also counterproductive if the result is that people give up and don't read anything.

If you want to evolve the help function so that it has some kind of "Show more..." link, then that's fine, but the current help box is intentionally as short as possible.

Reply:
Blackout help removed